packages/axelar-wasm-std-derive/src/lib.rs

```diff
@@ -282,3 +282,71 @@ fn match_unit_variant(event_enum: &Ident, variant_name: &Ident) -> TokenStream2
         #event_enum::#variant_name => cosmwasm_std::Event::new(#event_name)
     }
 }
+
+#[proc_macro_attribute]
+pub fn migrate_from_version(input: TokenStream, item: TokenStream) -> TokenStream {
+    let base_version_req = syn::parse_macro_input!(input as syn::LitStr).value();
```

Comment:
I'd much prefer parsing two separate inputs here `from_version` and `to_version` (both have to specify only major and minor) and forming the requirement here instead of having to handwrite that requirement myself.